Inserts.sql

```diff
@@ -1,16 +1,17 @@
 INSERT INTO Kunde (Name,Adresse) VALUES
-('Tom Meier','Raucherweg 44, 50667 K�ln'),
+('Tom Meier','Raucherweg 44, 50667 Köln'),
 ('Lisa Markov','Kondensatorstr. 11, 20097 Hamburg'),
 ('Enrica Fermi','Ableitung 2, 80336 Stuttgart');
 
 
 INSERT INTO Artikel (ArtikelNr,Bezeichnung,Preis) VALUES
-(99009,'Ladeger�t',23.90),
-(99102,'Kopfh�rer',145.99),
+(99009,'Ladegerät',23.90),
+(99102,'Kopfhörer',145.99),
 (99222,'Weihnachtskugel',5.00),
 (99700,'Krawatte',30),
 (99123,'Wecker',14.49),
-(99124,'Taschenkalender',5.99);
+(99124,'Taschenkalender',5.99)
+(99500, 'Taschenrechner', 9.99);
```

Comment:
Der ist aber Teuer. Hier ist ein vorschlag
```suggestion
(99500, 'Taschenrechner', 7.99);
```